frontend/src/components/posting/reply.js

```diff
@@ -96,6 +96,19 @@ export default class extends Form {
   }
 
   onCancel = () => {
+    const editorEmpty =
+      this.state.post.length === 0 && this.state.attachments.length === 0
+
+    // If only the quote text is on editor user didn't add anything
+    // so no changes to discard
+    const onlyQuoteTextInEditor =
```

Comment:
Seems flaky but still good enough for first iteration.

Reply:
I think for this I will add a quoteText class var where I will save the original replay and compare it to that is on the editor at the time of canceling.